tools/crm_attribute.c

```diff
@@ -98,14 +98,31 @@ struct {
     gchar *set_name;
     char *set_type;
     gchar *type;
+    enum pcmk__opt_table list_type;
     gboolean promotion_score;
 } options = {
     .command = 'G',
+    .list_type = pcmk__opt_table_cluster,
     .promotion_score = FALSE
 };
 
 #define INDENT "                               "
 
+static gboolean
+list_cb(const gchar *option_name, const gchar *optarg, gpointer data,
+        GError **error) {
+    options.command = 'L';
+
+    if (pcmk__str_eq(optarg, PCMK__VALUE_CLUSTER, pcmk__str_none)) {
+        options.list_type = pcmk__opt_table_cluster;
+    } else {
+        crm_warn("Invalid option type %s, using cluster options instead",
```

Comment:
TODO: This warning doesn't work because logging hasn't been initialized until after option processing.